docs/refs.bib

```diff
@@ -846,3 +846,25 @@ @misc{sykes2020boston
   title={- B 1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town},
   url={https://github.com/scikit-learn/scikit-learn/issues/16155}
 }
+
+@misc{wikibooks2010monotone,
+	title = {Algorithm Implementation/Geometry/Convex hull/Monotone chain},
+  author={Wikibooks},
+	howpublished = {\url{https://en.wikibooks.org/wiki/Algorithm_Implementation/Geometry/Convex_hull/Monotone_chain}},
+	year = {2010},
+	note = {[Accessed 09-Jul-2023]},
+}
+
+@misc{barocas2017tutorial,
```

Comment:
`@book{barocas2019fairness` sort of looks to be the same thing

Reply:
Agreed, I'll sub that reference instead.

Reply:
The book reference is mentioned earlier in that [ page](https://fairlearn.org/main/user_guide/further_resources.html), this reference is nested under the "Online demos, talks, tutorials" header.